src/App.js

```diff
@@ -37,50 +53,82 @@ class App extends React.Component {
         spoilerGuides: [],
       });
       axios
-        .post("https://imdb-parental-advisory.xsaudahmed.repl.co/findTitles", {
+        .post(`${API_URL}/findTitles`, {
           titleName: this.state.inputValue,
         })
         .then((res) => {
           this.setState({
             titleOptions: res.data,
             isLoading: false,
             noResultsFound: !res.data.length,
+            currentStep: STEPS.SELECT_TITLE,
           });
         });
     } else {
       this.setState({ errorMessage: "*Please enter a title" });
     }
   };
 
+  addTouchHandlers = () => {
+    document.querySelector("body").ontouchstart = (e) =>
+      this.setState({ touchStartX: e.changedTouches[0].clientX });
+
+    document.querySelector("body").ontouchend = (e) => {
+      if (
+        e.changedTouches[0].clientX - this.state.touchStartX >
+        SWIPE_THRESHOLD
+      ) {
+        this.goToSelectTitleStep();
+      }
+    };
+  };
+
+  removeTouchHandlers = () => {
+    document.querySelector("body").ontouchstart = null;
+    document.querySelector("body").ontouchend = null;
+  };
+
   GetParentalGuide = (titleId, titleSelection) => {
     this.setState({
       isLoading: true,
       selectedTitle: titleSelection,
     });
     axios
-      .post("https://imdb-parental-advisory.xsaudahmed.repl.co/parentalGuide", {
+      .post(`${API_URL}/parentalGuide`, {
         titleId,
       })
-      .then((res) =>
+      .then((res) => {
         this.setState({
           parentalGuides: res.data.parentalGuide.map(AddIdToSection),
           spoilerGuides: res.data.spoilersGuide.map(AddIdToSection),
-          isLoading: false,
-        })
-      );
+        });
+        setTimeout(
+          () =>
+            this.setState({
+              currentStep: STEPS.VIEW_GUIDES,
+              isLoading: false,
+            }),
+          250
+        );
```

Comment:
State is set in two parts to allow the next component to render off screen and then transition in. `setTimeout` allows us to delay the function by however long we decide.( In this case 250 ms.)